powerhome/playbook
# Use mutation observer to bind behavior of PbPopover

#### Runway Ticket URL

Make PbPopover behavior work with turbolinks navigation events. 

https://nitro.powerhrg.com/runway/backlog_items/GOG-83

#### Breaking Changes

Should be a transparent-to-the-client change to the way behavior is bound to the PbPopover so long as the client is importing the playbook webpack. 

Additional concerns would be the - hopefully unlikely case - that the client explicitly messes with the `window.PbPopover` which has been removed from the `vendor.js`.

#### How to Ninja test this (screenshots are really helpful)

Click on uses of the PbPopover kit. The behavior should not change.

<img width="1552" alt="Screen Shot 2020-02-20 at 7 53 56 AM" src="https://user-images.githubusercontent.com/59154/74935690-abd15f00-53b6-11ea-81ed-678dc9475e60.png">

#### Checklist:

- [x] **SPECS** Please cover your changes with specs (existing coverage)
